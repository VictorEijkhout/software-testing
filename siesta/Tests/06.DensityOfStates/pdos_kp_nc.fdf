%include ../base_fe.fdf
SystemLabel       pdos_kp_nc

Spin non-colinear
%block DM.InitSpin
    1 + 25.0 65.0
%endblock DM.InitSpin
Diag.ParallelOverK True

PDOS.kgrid.MonkhorstPack [10 10 10]
%block ProjectedDensityOfStates
EF -5.00 5.00 0.200000 1000 eV
%endblock ProjectedDensityOfStates

