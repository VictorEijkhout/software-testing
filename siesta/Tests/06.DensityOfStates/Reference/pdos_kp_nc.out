Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/06.DensityOfStates/work/pdos_kp_nc
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:36:32

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.86101
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel       pdos_kp_nc
Spin non-colinear
%block DM.InitSpin
    1 + 25.0 65.0
%endblock DM.InitSpin
Diag.ParallelOverK True
PDOS.kgrid.MonkhorstPack [10 10 10]
%block ProjectedDensityOfStates
EF -5.00 5.00 0.200000 1000 eV
%endblock ProjectedDensityOfStates
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: pdos_kp_nc
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   26 Label: Fe_soc


---- Processing specs for species: Fe_soc
Ground state valence configuration:   4s02  3d06
Reading pseudopotential information in PSML from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Fe_soc.psml
PSML file version: 1.1
Using libxc ids:  101 130
GGA--PBE XC_GGA_X_PBE--XC_GGA_C_PBE pb
PSML uuid: 89456970-44bf-11e8-4d23-a6a6edf19e30
Semicore shell(s) with  8 electrons included in the valence for Fe_soc

---- Pseudopotential check for Fe

Pseudopotential generated from a fully relativistic atomic calculation
There are spin-orbit semi-local pseudopotentials available

Pseudized shells:
3s( 2.00) rc: 1.26
3p( 6.00) rc: 1.21
3d( 6.00) rc: 1.57
Valence configuration for ps generation: 3s:3p:3d:4s: Total charge:  16.000000
Semicore shell(s): 3s 3p
Fallback to non-perturbative polarization scheme for Fe_soc
Please see the manual if this is not desired in principle
For Fe_soc, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
Fe_soc pseudopotential only contains V_ls up to l=2 -- lmxkb reset.

<basis_specs>
===============================================================================
Fe_soc               Z=  26    Mass=  55.845        Charge= 0.17977+309
Lmxo=2 Lmxkb= 2    BasisType=split      Semic=T
L=0  Nsemic=1  Cnfigmx=4
          i=1  nzeta=1  polorb=0  (3s)  (semicore shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
          i=2  nzeta=1  polorb=0  (4s)  (to be polarized non-perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=1  Cnfigmx=4
          i=1  nzeta=1  polorb=0  (3p)  (semicore shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
          i=2  nzeta=1  polorb=0  (4p)  (non-perturbative polarization shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:   0.69471-309
               qyuk:   0.16956-319
               qwid:   0.69526-309
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=2  erefs: 0.17977+309 0.17977+309
L=1  Nkbl=2  erefs: 0.17977+309 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Fe_soc                (Z =  26)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ONCVPSP-3.Hamann's oncvpsp                        
Valence charge in pseudo generation:   16.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
Got Vlocal (oncv-fit) from psml data
Choosing vlocal chloc cutoff:  3.831373
qtot up to nchloc:   15.99793860
atom: Maximum radius for chlocal:    3.83137
atom: Maximum radius for r*vlocal+2*Zval:    3.83137
Reading KB projs from Fe_soc psml data
Using (derived) scalar-relativistic projectors in PSML file

PSML: Kleinman-Bylander projectors: 
   l= 0   rc=  1.280242   Ekb=  2.929588
   l= 0   rc=  1.280242   Ekb= 20.336184
   l= 1   rc=  1.221004   Ekb=  3.212733
   l= 1   rc=  1.221004   Ekb= -0.970660
   l= 1   rc=  1.221004   Ekb=  0.001568
   l= 1   rc=  1.221004   Ekb= -0.000083
   l= 2   rc=  1.586526   Ekb=-11.013454
   l= 2   rc=  1.586526   Ekb= -4.065821

PSML: Total number of  Kleinman-Bylander projectors:  24
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    2.378892
                 energy =   -6.894702
                kinetic =    3.214823
    potential(screened) =  -10.109525
       potential(ionic) =  -37.547812

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.575234
                 energy =   -0.437248
                kinetic =    1.185569
    potential(screened) =   -1.622817
       potential(ionic) =  -14.647486

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    2.635420
                 energy =   -4.398765
                kinetic =    7.852272
    potential(screened) =  -12.251038
       potential(ionic) =  -38.913597

SPLIT: Basis orbitals for state 4p
SPLIT: Non-perturbative polarization orbs for state 4s
SPLIT: rc(1st zeta) (re)set to: 6.5752 (expansion factor: 1.0000)
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.575234
                 energy =   -0.021587
                kinetic =    1.092964
    potential(screened) =   -1.114551
       potential(ionic) =  -11.623296

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.097563
                 energy =   -0.539341
                kinetic =   10.289951
    potential(screened) =  -10.829292
       potential(ionic) =  -34.384996
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00) 4s( 2.00)                                                  
 3p( 6.00) 4p( 0.00)                                                  
 3d( 6.00)                                                            
Vna: chval, zval:   16.00000  16.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.575234
comcore: Pseudo-core radius Rcore=  1.729904

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   26 Fe_soc                  # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Fe_soc                5                    # Species label, number of l-shells
 n=3   0   1                         # n, l, Nzeta 
   2.379   
   1.000   
 n=4   0   1                         # n, l, Nzeta 
   6.575   
   1.000   
 n=3   1   1                         # n, l, Nzeta 
   2.635   
   1.000   
 n=4   1   1                         # n, l, Nzeta 
   6.575   
   1.000   
 n=3   2   1                         # n, l, Nzeta 
   4.098   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

CH_OVERLAP: New q2cut (Ry) =   3100.00
CH_OVERLAP: New q2cut (Ry) =   3844.00
CH_OVERLAP: New q2cut (Ry) =   4766.56
CH_OVERLAP: New q2cut (Ry) =   5910.53
Dumping basis to NetCDF file Fe_soc.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      1    13    24

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = non-collinear
redata: Number of spin components                   = 4
redata: Time-Reversal Symmetry                      = F
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       50
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   116.0452 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 10
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
mix.SCF: Spin-component mixing                   all
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 10
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000

Time-reversal symmetry not used.

siesta: k-grid: Number of k-points = 343
siesta: k-points from Monkhorst-Pack grid
siesta: k-cutoff (effective) =     8.438 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     7     0     0      0.000
siesta: k-grid:     0     7     0      0.000
siesta: k-grid:     0     0     7      0.000
k-point displ. along   1 input, could be:     0.00    0.50
k-point displ. along   2 input, could be:     0.00    0.50
k-point displ. along   3 input, could be:     0.00    0.50

Time-reversal symmetry not used.

siesta: PDOS k-grid: Number of k-points = 1000
siesta: PDOS k-points from Monkhorst-Pack grid
siesta: PDOS k-cutoff (effective) =    12.054 Ang
siesta: PDOS k-point supercell and displacements
siesta: k-grid:    10     0     0      0.000
siesta: k-grid:     0    10     0      0.000
siesta: k-grid:     0     0    10      0.000

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     9 x     9 x     9  =     729
superc: Number of atoms, orbitals, and projectors:    729   9477  17496


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     9 x     9 x     9  =     729
superc: Number of atoms, orbitals, and projectors:    729   9477  17496

outcell: Unit cell vectors (Ang):
       -1.391873    1.391873    1.391873
        1.391873   -1.391873    1.391873
        1.391873    1.391873   -1.391873

outcell: Cell vector modules (Ang)   :    2.410795    2.410795    2.410795
outcell: Cell angles (23,13,12) (deg):    109.4712    109.4712    109.4712
outcell: Cell volume (Ang**3)        :     10.7860
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=13 nrows=13 sparsity=65.9822 nnzs=11151, refcount: 7>
  <dData1D:(new from dSpData1D) n=11151, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data (user-defined):
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=13 nrows=13 sparsity=65.9822 nnzs=11151, refcount: 8>
  <dData2D:DM n=11151 m=4, refcount: 1>
refcount: 1>
     spin moment: {S} , |S| = {    0.71442    1.53209    3.62523 }     4.00000
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      27     187

InitMesh: MESH = 24 x 24 x 24 = 13824
InitMesh: Mesh cutoff (required, used) =   300.000   410.861 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -606.271927
siesta: Eions   =      4922.283762
siesta: Ena     =       584.711018
siesta: Ekin    =      1545.761883
siesta: Enl     =      -184.223200
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        48.334162
siesta: DUscf   =         1.442279
siesta: DUext   =         0.000000
siesta: Ex      =      -455.255111
siesta: Ec      =       -26.304021
siesta: Exc     =      -481.559132
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -3399.369185
siesta: Etot    =     -3407.816753
siesta: FreeEng =     -3407.817567

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1    -3399.369185    -3407.816753    -3407.817567  0.903698 -2.266506  6.337907
     spin moment: {S} , |S| = {    0.49177    1.05461    2.49543 }     2.75340
timer: Routine,Calls,Time,% = IterSCF        1       0.395  29.93
   scf:    2    -3407.864006    -3407.840429    -3407.841310  0.002359 -2.299659  6.223982
     spin moment: {S} , |S| = {    0.49123    1.05344    2.49265 }     2.75033
   scf:    3    -3408.858764    -3408.531808    -3408.532414  0.115201 -4.291450  0.771026
     spin moment: {S} , |S| = {    0.45198    0.96928    2.29351 }     2.53061
   scf:    4    -3407.886071    -3408.261924    -3408.262397  0.048660 -5.183072  3.824475
     spin moment: {S} , |S| = {    0.45198    0.96928    2.29351 }     2.53061
   scf:    5    -3408.575617    -3408.453265    -3408.455074  0.040117 -4.252334  1.353427
     spin moment: {S} , |S| = {    0.47054    1.00907    2.38766 }     2.63449
   scf:    6    -3408.807172    -3408.621595    -3408.622089  0.074698 -4.182033  0.345103
     spin moment: {S} , |S| = {    0.40199    0.86208    2.03985 }     2.25073
   scf:    7    -3408.608122    -3408.615689    -3408.616183  0.006529 -4.015119  0.724811
     spin moment: {S} , |S| = {    0.40199    0.86208    2.03985 }     2.25073
   scf:    8    -3408.612061    -3408.614236    -3408.614908  0.007236 -4.186885  0.488450
     spin moment: {S} , |S| = {    0.40623    0.87116    2.06135 }     2.27444
   scf:    9    -3408.632669    -3408.623683    -3408.624176  0.008863 -4.209079  0.130797
     spin moment: {S} , |S| = {    0.40199    0.86208    2.03985 }     2.25073
   scf:   10    -3408.584030    -3408.602916    -3408.602955  0.016541 -4.158612  0.527952
     spin moment: {S} , |S| = {    0.41453    0.88895    2.10344 }     2.32089
   scf:   11    -3408.539265    -3408.569750    -3408.570219  0.016703 -4.059178  0.860587
     spin moment: {S} , |S| = {    0.43011    0.92237    2.18252 }     2.40815
   scf:   12    -3408.534069    -3408.551730    -3408.552457  0.007223 -4.054142  1.066565
     spin moment: {S} , |S| = {    0.43570    0.93435    2.21087 }     2.43943
   scf:   13    -3408.712258    -3408.624003    -3408.624496  0.041209 -4.135141  0.232759
     spin moment: {S} , |S| = {    0.40199    0.86208    2.03985 }     2.25073
   scf:   14    -3408.643313    -3408.630937    -3408.631027  0.024032 -4.155236  0.224294
     spin moment: {S} , |S| = {    0.38293    0.82119    1.94311 }     2.14399
   scf:   15    -3408.632469    -3408.631748    -3408.631757  0.002936 -4.130545  0.084192
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
   scf:   16    -3408.631719    -3408.631733    -3408.631746  0.000923 -4.111801  0.073651
     spin moment: {S} , |S| = {    0.38323    0.82183    1.94462 }     2.14566
   scf:   17    -3408.631947    -3408.631851    -3408.631860  0.002331 -4.120244  0.018929
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
   scf:   18    -3408.631857    -3408.631854    -3408.631863  0.000152 -4.123349  0.009847
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
   scf:   19    -3408.631856    -3408.631855    -3408.631864  0.000059 -4.123149  0.007372
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
mix: Pulay -- inversion failed, > SVD [rank/size] 7 / 9
   scf:   20    -3408.631839    -3408.631847    -3408.631856  0.000315 -4.128911  0.015921
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94474 }     2.14578
mix: Pulay -- inversion failed, > SVD [rank/size] 7 / 9
   scf:   21    -3408.631839    -3408.631843    -3408.631852  0.000109 -4.129622  0.019885
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94474 }     2.14578
   scf:   22    -3408.631848    -3408.631843    -3408.631852  0.001363 -4.113177  0.022723
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
mix: Pulay -- inversion failed, > SVD [rank/size] 6 / 9
   scf:   23    -3408.631871    -3408.631857    -3408.631866  0.000700 -4.121096  0.000213
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
   scf:   24    -3408.631857    -3408.631857    -3408.631866  0.000049 -4.120839  0.001746
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577
mix: Pulay -- inversion failed, > SVD [rank/size] 7 / 9
   scf:   25    -3408.631857    -3408.631857    -3408.631866  0.000042 -4.121092  0.000004
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000418377
max |H_out - H_in|      (eV) :     0.0000044058
SCF cycle converged after 25 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      27     187

siesta: E_KS(eV) =            -3408.6319

siesta: E_KS - E_eggbox =     -3408.6319
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000   -0.000000    0.000000
----------------------------------------
   Max    0.000000
   Res    0.000000    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000000    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):     -193.90     -193.90     -193.90        0.27        0.27        0.27
(Free)E + p*V (eV/cell)    -3407.3265
Target enthalpy (eV/cell)    -3408.6319

siesta: PDOS info
siesta: Shifting energies with respect to Fermi-level
siesta: E1 -- E2, sigma [eV], nhist:   -9.121 --    0.879    0.200 1000

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -576.191163
siesta: Eions   =      4922.283762
siesta: Ena     =       584.711018
siesta: Ekin    =      1579.434900
siesta: Enl     =      -193.103056
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        27.880342
siesta: DUscf   =         0.427243
siesta: DUext   =         0.000000
siesta: Ex      =      -459.321344
siesta: Ec      =       -26.377197
siesta: Exc     =      -485.698542
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -3408.631857
siesta: Etot    =     -3408.631857
siesta: FreeEng =     -3408.631866

siesta: Final energy (eV):
siesta:  Band Struct. =    -576.191163
siesta:       Kinetic =    1579.434900
siesta:       Hartree =     819.793649
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -459.321344
siesta:         Corr. =     -26.377197
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -485.698542
siesta:  Ion-electron =   -3307.053249
siesta:       Ion-ion =   -2015.108614
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =   -3408.631857
siesta:         Fermi =      -4.121092

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.121022    0.000166    0.000166
siesta:     0.000166   -0.121022    0.000166
siesta:     0.000166    0.000166   -0.121022

siesta: Cell volume =         10.785960 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00131809          0.00131809  Ry/Bohr**3
siesta:           0.12102160          0.12102160  eV/Ang**3
siesta:         193.89798583        193.89798583  kBar
(Free)E+ p_basis*V_orbitals  =       -3408.113414
(Free)Eharris+ p_basis*V_orbitals  =       -3408.113414
     spin moment: {S} , |S| = {    0.38325    0.82188    1.94473 }     2.14577

cite: Please see "pdos_kp_nc.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =      12.788
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1      12.768      12.768   100.00
Setup                  1       0.071       0.071     0.55
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1      12.197      12.197    95.53
geom_init              1       0.853       0.853     6.68
state_init             1       0.156       0.156     1.22
hsparse                1       0.002       0.002     0.02
overlap                1       0.053       0.053     0.42
Setup_H0               1       0.529       0.529     4.14
naefs                  2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.027       0.053     0.42
nlefsm                 2       0.176       0.352     2.76
DHSCF_Init             1       0.158       0.158     1.23
DHSCF1                 1       0.022       0.022     0.17
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.135       0.135     1.06
REMESH                 1       0.025       0.025     0.20
REORD                236       0.000       0.003     0.03
PHION                  1       0.095       0.095     0.74
COMM_BSC             438       0.000       0.004     0.03
POISON                28       0.000       0.006     0.05
fft                   56       0.000       0.004     0.03
setup_H               26       0.175       4.557    35.69
DHSCF                 27       0.209       5.634    44.12
DHSCF3                27       0.176       4.752    37.22
rhoofd                27       0.082       2.213    17.33
XC                    27       0.010       0.273     2.14
GXC-CellXC            27       0.010       0.271     2.12
gridxc@cellXC         27       0.010       0.271     2.12
vmat                  27       0.083       2.249    17.62
IterSCF               25       0.409      10.228    80.11
compute_dm            25       0.232       5.790    45.35
diagon                25       0.232       5.789    45.34
cdiag              18150       0.000       0.955     7.48
cdiag1             18150       0.000       0.039     0.30
cdiag2             18150       0.000       0.185     1.45
cdiag3             18150       0.000       0.635     4.97
cdiag4              9575       0.000       0.032     0.25
MIXER                 24       0.001       0.032     0.25
PostSCF                1       1.115       1.115     8.73
DHSCF4                 1       0.882       0.882     6.91
dfscf                  1       0.864       0.864     6.76
overfsm                1       0.002       0.002     0.01
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.500       0.500     3.92
pdos                   1       0.497       0.497     3.89
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:36:45
Job completed
